services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/gdpr/KortistoGdprServiceITest.kt

```diff
@@ -187,13 +187,14 @@ class KortistoGdprServiceITest(
                     ytunnus = "3213212-0",
                 )
             val hanke = hankeFactory.saveMinimal(generated = true)
```

Comment:
Not actually part of this PR, but it caught my eye: `HankeFactory.saveMinimal` does not create a `HankeKayttaja` as e.g. `HankeFactoty.saveWithAlue(USERNAME)` does. Should we provide this kind of Hanke creation that does not create a `HankeKayttaja` at all but instead make `saveMinimal` to create `HankeKayttaja`? 

I made this comment on this PR because I wondered why there are different ways to create `hanke` and `kayttaja` in these tests here.

Reply:
Sometimes we want to create minimalistic data for tests. Like, more minimalistic than could be created with production code. In some cases, we need to test what happens when there is no user, and in others, it's clearer if the user is created separately. Having a convenient way to create those is better than forcing every test that needs something like that to call the repository directly.